app/controllers/api/pods_controller.rb

```diff
@@ -113,8 +113,17 @@ def verify_pushes_allowed!
                       'your version of CocoaPods to push this specification.'
             json_error(422, message)
           end
+
+          if Version.new(version[1]) == Version.new("1.3.0") 
```

Comment:
@orta should we be checking if the spec actually contains test specifications? We could allow releases of all specs without test specs